misaligned.py

```diff
@@ -1,13 +1,15 @@
-
+global last_pair_number;
 def print_color_map():
     major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
             print(f'{i * 5 + j} | {major} | {minor}')
-    return len(major_colors) * len(minor_colors)
-
+            # returning last pairnumber to check if it's 25
+    return len(major_colors) * len(minor_colors),i * 5 + j
 
-result = print_color_map()
+result,last_pair_number = print_color_map()
 assert(result == 25)
+# including testcase to check if its following 1 based index
+assert(last_pair_number == 25)

```

Comment:
The number 5 used in the pair number calculation would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.